smother/control.py

```diff
@@ -119,9 +131,25 @@ def load(cls, file_or_path):
         result.data = data
         return result
 
+    def _normalize_path(self, path):
+        aliases = None
+        if self.coverage and self.coverage.config.paths:
+            aliases = PathAliases()
+            for paths in self.coverage.config.paths.values():
+                result = paths[0]
+                for pattern in paths[1:]:
+                    aliases.add(pattern, result)
+            path = aliases.map(path)
+
+        if self.relative_paths:
+            set_relative_directory()
```

Comment:
https://bitbucket.org/ned/coveragepy/src/7c3dddfe83893c51035c0be4f35b58d0379858ac/coverage/files.py?at=default&fileviewer=file-view-default

Sets some icky globals that are used to create the relative path int he next line. Not the best, but already well tested since it's from coverage.